src/common/react-novnc.js

```diff
@@ -86,14 +85,17 @@ export default class NoVnc extends Component {
     const rfb = this._rfb = new RFB({
       encrypt: isSecure,
       target: this.refs.canvas,
-      wsProtocols: [ 'chat' ],
+      wsProtocols: [ 'binary' ],
```

Comment:
it is not necessary and it is the new default https://github.com/novnc/noVNC/blob/master/core/rfb.js#L157